libnavigation-router/src/test/java/com/mapbox/navigation/route/internal/RouterWrapperTests.kt

```diff
@@ -264,9 +264,7 @@ class RouterWrapperTests {
                     .toUrl(),
                 routerOrigin = Onboard,
                 message = "failed for response: ${routerResultSuccessErroneousValue.value}",
-                throwable = IllegalStateException(
-                    """route response should contain "routes" array"""
-                )
+                throwable = IllegalStateException("Null routes")
```

Comment:
@LukasPaczos is there any reason to be strict in the message?

Reply:
Not really.